boughtbymany/mutt-forms-vue
# i18n Fixes

# Description

Fixes missing select field options text and missing error message text. I have also checked for other occurrences of the issue.

This also updates the demo form to validate when submitted.

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Use the demo app and follow the steps below to test:

## Validation message
1. Delete the text content of `Text Field`
1. Click **Submit/Vorlegen**
1. The validation message 'This field is required.' should be displayed under `Text Field`

## Select field
1. Click `Choice Field`
1. The options `FIRST` and `SECOND` should be displayed.

# Checklist:
